core/src/main/java/io/fabric8/maven/core/util/kubernetes/KindFilenameMapperUtil.java

```diff
@@ -0,0 +1,39 @@
+package io.fabric8.maven.core.util.kubernetes;
+
+import io.fabric8.maven.core.util.AsciiDocParser;
+import io.fabric8.maven.core.util.EnvUtil;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class KindFilenameMapperUtil {
+
+    public static String[] loadMappings() {
+
+        final String location =
+            EnvUtil.getEnvVarOrSystemProperty("fabric8.mapping", "/fabric8/default-kind-filename-mapping.adoc");
```

Comment:
We already store some meta data in the classpath, but in `META-INF/fabric8` (like the `enrichers-default`). Could we switch the location to `META-INF/fabric8`, too ? And maybe use the `kind-fileindicator-mapping-default.adoc` as name ? (to adhere to the current convention to add `-default` towards the end).